packages/honeycomb-opentelemetry-web/rollup.config.mjs

```diff
@@ -1,33 +1,17 @@
-import { createRequire } from 'node:module';
 import { DEFAULT_EXTENSIONS } from '@babel/core';
 import analyze from 'rollup-plugin-analyzer';
-import babel from '@rollup/plugin-babel';
+import { babel } from '@rollup/plugin-babel';
 import commonjs from '@rollup/plugin-commonjs';
 import dts from 'rollup-plugin-dts';
-import nodeResolve from '@rollup/plugin-node-resolve';
+import { nodeResolve } from '@rollup/plugin-node-resolve';
 import typescript from 'rollup-plugin-typescript2';
 import autoExternal from 'rollup-plugin-auto-external';
 
-const require = createRequire(import.meta.url);
-const pkg = require('./package.json');
-
-// get dependencies and peer dependencies as declared in package.json
-const getExternalDepsFromPackageJSON = () => {
```

Comment:
Do we not need this anymore? It was unused?

Reply:
When I did this originally I wrote this code and then realized I could use `rollup-plugin-auto-external` and _because_ this wasn't being linted, the dead code just was chillin.'